examples/react-es6/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "react": "0.13.0-beta.1",
+    "react-tools": "0.13.0-beta.1"
+  },
+  "devDependencies": {
+    "babel-jest": "*"
```

Comment:
Let's not use babel-jest and just do what we do for the other React example. The preprocessor would just want to `ReactTools.transform(src, {harmony: true}}` to get classes working.